Wiley Online Library.js

```diff
@@ -255,6 +255,11 @@ function scrapeBibTeX(doc, url, pdfUrl) {
 				item.title = ZU.capitalizeTitle(item.title, true);
 			}
 			
+			//date in the cochraine library RIS is wrong
+			if (ZU.xpathText(doc, '//meta[@name="citation_book_title"]/@content') == "The Cochrane Library") {
```

Comment:
let's also test here for the presence of citation_online_date, just to be extra safe. 

(and I don't recall, but testing for an exact match of null--i.e. the case that there is no citation_book_title--doesn't throw an error, right?)